src/fs/save.rs

```diff
@@ -43,17 +43,8 @@ pub trait PersistentData: DeserializeOwned + Serialize {
         let load_result = <T as PersistentData>::load();
 
         match load_result {
-            Err(ClientError::LoadFileDoesntExist(_)) => Ok(get_default_and_save()?),
-            Err(ClientError::YamlError(error)) => {
-                // Prevent parsing error with an empty string and commented out file.
-                if error.to_string() == "EOF while parsing a value" {
-                    Ok(get_default_and_save()?)
-                } else {
-                    Err(ClientError::YamlError(error))
-                }
-            }
-            Err(error) => Err(error),
             Ok(deser) => Ok(deser),
+            _ => Ok(get_default_and_save()?),
```

Comment:
If anything goes wrong loading fingerprint.yml I really just want us to fallback to default rather than throwing error.

Reply:
I think this is fine for our persistent data trait since nothing should be touched by the end user. If we had a user controlled config file, this would be dangerous to do since we could easily overwrite their changes.